### DISCUSSION TITLE
建议在开源的前提下，和厂家合作，推出团购版。

### DISCUSSION THREAD

**Discussion by greatbody** [peng-zhihui/HelloWord-Keyboard#26] [Ideas]
建议在开源的前提下，和厂家合作，推出团购版。

开模，等等由商家进行。稚晖君进行品牌授权（注册稚晖君品牌），并进行抽样质量检测。

声明团购定义为大家共同委托厂家进行定制化生产，收货一周内有硬件质量问题可退换，除此外，不提供任何保修。

这样，大家可以更快的拿到稚晖君的新品，而且稚晖君也能培育自己的品牌，同时不用消耗太多的精力。

**Comment by peng-zhihui** [comment]
我不卖东西也不靠这个盈利的，这个项目初衷只是满足我个人的使用需求，顺便开源出来而已。当然很高兴很多人都喜欢这个作品，只是大家开团要擦亮眼睛不要被某些JS给骗了，我没有特别授权给任何商家，只是同意大家使用我的开源方案；如果确实对开团有兴趣，建议关注xikii的开团计划，毕竟这个键盘本体就是他设计的，我也是基于他的设计做了改造。

**Reply by lovecoding2023** [reply]
我们是专业做塑胶产品样品打印和小批量制作的厂家，服务过全球很多知名硬件项目，可以使用硅胶模具的方式制作出接近注塑的产品，比打印件更耐用美观，同时确保低成本，对稚晖君的这个项目很感兴趣，想作为一个新方向的尝试，如果稚晖君不反对，有兴趣开团塑胶件的小伙伴可以联系我QQ 164488 :)

**Comment by greatbody** [comment]
提这个建议，主要是想方便粉丝们降低获取的门槛，同时也希望避免无良JS打着稚晖君的名号骗钱。

不知道 xikii 是 Github 的账号，还是什么别的平台的账号？这样方便粉丝们联系？

感谢，祝好。

**Reply by CC52242** [reply]
bili，xikii厂长
